### PR TITLE
Convert "Info and Settings" tab on ManageEvent to use datepicker

### PR DESCRIPTION
**Pull request by mattwire**
Overview
----------------------------------------
Convert "Info and Settings" tab on ManageEvent to use datepicker.

Before
----------------------------------------
Used old jcalendar widget

After
----------------------------------------
Uses datepicker widget

Technical Details
----------------------------------------
The first commit is formatting cleanup only for the template.  The second commit contains the functional changes.

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Event/Form/ManageEvent/EventInfo.php [hunk 1]
```diff
@@ -177,8 +172,8 @@ public function buildQuickForm() {
     $this->addElement('checkbox', 'is_share', ts('Allow sharing through social media?'));
     $this->addElement('checkbox', 'is_map', ts('Include Map to Event Location'));
 
-    $this->addDateTime('start_date', ts('Start Date'), FALSE, array('formatType' => 'activityDateTime'));
-    $this->addDateTime('end_date', ts('End Date / Time'), FALSE, array('formatType' => 'activityDateTime'));
+    $this->add('datepicker', 'start_date', ts('Start Date'), [], FALSE, ['time' => TRUE]);
+    $this->add('datepicker', 'end_date', ts('End Date / Time'), [], FALSE, ['time' => TRUE]);
```
Comment (colemanw) [inline CRM/Event/Form/ManageEvent/EventInfo.php]: Scope creep: these form labels are inconsistent. Why mention time in one but not the other?
What about simply "Start" and "End"?

**Comment by colemanw** [comment]
Tested and works well. Giving MOP label for whenever our testbot comes back from the dead :face_with_head_bandage:

**Comment by eileenmcnaughton** [comment]
argh @mattwire this needs rebasing

**Comment by seamuslee001** [comment]
Test fails unrelated, Merging as per the tag
